src/components/AuthNav/AuthNav.tsx

```diff
@@ -6,24 +6,36 @@ import { buttonVariant } from 'components/ui/Button';
 
 import styles from './AuthNav.module.css';
 
-const AuthNav = () => {
+interface AuthNavProps {
+  onClose: () => void;
+}
+
+const AuthNav = (props: AuthNavProps) => {
+  const { onClose } = props;
+  const isSmallScreen = window.innerWidth < 1194;
+
+  const handleButtonClick = () => {
+    if (onClose) {
+      onClose();
+    }
+  };
```

Comment:
Why you call this props onClose. It's looks like onClick to menu item.
And you can just pass it in `<Link onClick={onClick} >`